### PR TITLE
Configured Swagger codegen maven plugin and setup.

### PR DESCRIPTION
**Pull request by instanceof-inf**

### CODE REVIEW

File: src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/ApplicationConfiguration.java
Comment (RossVanDerHeyde) [inline src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/ApplicationConfiguration.java:1]: Why was this file removed?

Comment (RossVanDerHeyde) [reply src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/ApplicationConfiguration.java:1]: I'd suggest kepping it in as it will be needed for future tasks

File: pom.xml [hunk 1]
```diff
@@ -46,6 +139,68 @@
...
+					<execution>
+						<id>add-source</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>add-source</goal>
+						</goals>
+						<configuration>
+							<sources>
+								<source>
+									TODO
```
Comment (RossVanDerHeyde) [inline pom.xml:128]: Looks like the `configuation` tag isn't doing much, it could likely be removed.

File: pom.xml [hunk 2]
```diff
@@ -33,6 +39,93 @@
...
+		</dependency>
+
+		<dependency>
+			<groupId>io.swagger</groupId>
+			<artifactId>swagger-annotations</artifactId>
+			<version>1.5.21</version>
+		</dependency>
+
+		<!-- HTTP client: jersey-client -->
+		<dependency>
```
Comment (RossVanDerHeyde) [inline pom.xml]: There are a couple dependencies in this file that don't seem necessary, e.g. this one, migbase64.
Please remove them, just keep in what you need for now.

File: src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/ApplicationConfiguration.java
Comment (instanceof-inf) [reply src/main/java/com/johan/blignaut/entelect/vitalityincubator/task1/ApplicationConfiguration.java:1]: It was removed because it is no longer needed because of the swagger generated files doing it's job. I will add it back in when necessary, because at the moment it contains no code.

File: pom.xml (hunk 1 above)
Comment (instanceof-inf) [reply pom.xml:128]: This was from the code you gave me, I didn't change anything. I also just tried to remove it, but then getting an error saying `The parameters 'sources' for goal org.codehaus.mojo:build-helper-maven-plugin:3.0.0:add-source are missing or invalid`, so leaving it in for now, is that okay?

Comment (RossVanDerHeyde) [reply pom.xml:128]: I'm not sure how it's working without a valid directory here😅. This should be the package where the generated code lives.

**Review by vincentreid — APPROVED** [review]

**Review by MrCaseys — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (instanceof-inf) [reply pom.xml:128]: I'm not sure either haha, but if I take it out, it breaks. Like I said, this was exactly copied from the code you gave me.

**Review by RossVanDerHeyde — APPROVED** [review]
